Product/Production/Gateway/PatientDiscovery_10/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/_10/servlet/InitServlet.java

```diff
@@ -80,10 +80,11 @@
     @SuppressWarnings("static-access")
     public void init(ServletConfig config) throws ServletException {
         LOG.debug("InitServlet start...");
-        executor = Executors.newFixedThreadPool(ExecutorServiceHelper.getInstance().getExecutorPoolSize());
-        largeJobExecutor = Executors.newFixedThreadPool(ExecutorServiceHelper.getInstance()
+        ExecutorService newExecutor = Executors.newFixedThreadPool(ExecutorServiceHelper.getInstance().getExecutorPoolSize());
+        setExecutorService(newExecutor);
+        ExecutorService newLargeJobExecutor = Executors.newFixedThreadPool(ExecutorServiceHelper.getInstance()
```

Comment:
Question: In some InitServletClass, we use ExecutorService and some are not.

Reply:
This change was purely to get rid of 2 "critical" sonar violations. I would have left it the way it was.